src/System.Device.Gpio/System/Device/Gpio/Drivers/Windows10DriverPin.Windows.cs

```diff
@@ -12,7 +12,7 @@ namespace System.Device.Gpio.Drivers
 {
     internal class Windows10DriverPin : IDisposable
     {
-        private const int ReasonableDebounceTimeoutMillseconds = 50;
+        private const int ReasonableDebounceTimeoutMilliseconds = 0;
```

Comment:
50ms is way to long if you want to count events. This may be feasible for stuff like buttons, but not for digital inputs. This change makes the Windows build pass the fast event test.

Reply:
Note: If we want a bounce timeout feature, we should implement it globally, not in one place only (and different from the other drivers)